README.md

```diff
@@ -61,4 +61,28 @@ $ make run
 ```
 
 You should be able to test the forum at [http://localhost:8090/dev/](http://localhost:8090/dev/).
-This is the development view that emulates a LTI consumer.
\ No newline at end of file
+This is the development view that emulates a LTI consumer.
+
+## Backoffice
+
+You can access the backoffice of the forum at [http://localhost:8090/admin/](http://localhost:8090/admin/)
+
+You first need to create a superuser account. 
+
+```bash
+$ bin/superuser
+```
+
+## Troubleshooting
+
+ElasticSearch service is always down with an Exit 137 State
```

Comment:
We might have multiple solutions to provide in this section, so I would add this in a sub-section  : 

```suggestion
### ElasticSearch service is always down with an Exit 137 State
```

Reply:
Yes but it's a little bit long for a title. Maybe just keeping `ElasticSearch service is always down`

Reply:
Regarding the error in Richie I thought it would be good to specify Exit 137 as it surprised us not to find anything in logs

Reply:
It is now ### ElasticSearch service out with an Exit 137
hope it's good to everybody